src/content/smart-bulletins/031.md

```diff
@@ -0,0 +1,103 @@
+
+Effective upon issuance and shall remain in force until modified,
+rescinded, or superseded.
+
+## Business Line(s) Affected
+
+Purchase, Travel, Fleet and Integrated
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide implementation guidance, updates and clarifications relating to portions of OMB Circular A-123, Appendix B, ‘A Risk Framework for Charge Card Programs (“the Circular”). These revisions were coordinated with the Office of Management and Budget, Office of Federal Financial Management.
+
+## Summary
+
+This Smart Bulletin addresses issues regarding creditworthiness, sustainability, environmental sustainability, definition of terms and other matters.
+
+### Chapter 5 – Performance Metrics and Data Requirements
+Section 5.3.1 implementing statistical reporting is updated as follows: 
+
+The reference in this section to the GSA SmartPay website shall be read to refer to the GSA Interact, the secure platform used to post agency-specific metrics.
+### Chapter 6 – Credit Worthiness for Individually Billed Accounts (IBAs)
+Section 6.4 implementing guidance is updated as follows: 
+
+If it is not possible to obtain a card applicant’s credit score, the agency may issue a “restricted” charge card (as defined in section 6.3.2.,”First time travel charge card applicants with a credit score of less than 660”) to a first time applicant, but the agency must ensure it first conducts an alternative creditworthiness assessment to determine whether the individual possesses a satisfactory credit history. Specifically, the agency must review the applicant’s most recent Standard Form (SF) 85P, Questionnaire for Public Trust Positions, Section 22, or SF 86, Section 26, or use a similar vehicle containing the same type of questions as in the forms and sections noted and use the information provided to assess creditworthiness. In either case, the vehicle used must not be older than one year. 
+
+**Note:** Neither the SF 86 nor SF 26 is to be shared with your servicing contractor bank.
+
+Section 6.11 is clarified as follows: 
+
+The reference to the “OPM Center for Federal Investigative Services (FIS)” is hereby deleted and replaced with “Defense Counterintelligence and Security Agency (DCSA).”
+
+Section 6.12 is clarified as follows:
+
+The reference to GSA’s Financial and Business Solutions (FABS) Schedule contractors and its associated footnote is hereby deleted and replaced with “PS-MAS.” More information on PS-MAS (Professional Services – Multiple Award Schedule) in this context can be found at: gsa.gov/psschedules, under “professional services/identity protection services.”
+
+### Chapter 10 – Sustainability Environmental Requirements
+Due to the issuance of Executive Order (EO) 13834, “Efficient Federal Operations,” dated May 17, 2018, and the revocation of EO 13693, “Planning for Federal Sustainability in the Next Decade,” dated March 19, 2015, Section 10.2 is deleted in its entirety are replaced with the following: 
```

Comment:
Due to the issuance of Executive Order (EO) 13834, “Efficient Federal Operations,” dated May 17, 2018, and the revocation of EO 13693, “Planning for Federal Sustainability in the Next Decade,” dated March 19, 2015, Section 10.2 is deleted in its entirety are replaced with the following:
EO 13834 affirms that it is the policy of the United States that agencies meet energy and environmental performance statutory requirements in a manner that increases efficiency, optimizes performance, eliminates unnecessary use of resources, and protects the environment. In implementing this policy, agencies are tasked to prioritize actions that reduce waste, cut costs, enhance the resilience of Federal infrastructure and operations, and enable more effective accomplishment of its mission. EO 13834 may be viewed at: https://www.fedcenter.gov/programs/eo13834.
To assist agencies in meeting statutory and optional sustainable purchasing requirements, GSA offers an online Green Procurement Compilation. The Green Procurement Compilation is a comprehensive green purchasing resource designed for federal contracting personnel and program managers. It consolidates and organizes information from federal environmental programs in one place, saving you from visiting multiple websites.
The Green Procurement Compilation allows users to:
Quickly identify federal green purchasing requirements for the products and services;
Search by keyword or browse by category to find products and services;
Determine procurement options available to federal buyers, including applicable GSA Multiple Award Schedules (MAS), Federal Strategic Sourcing Initiative (FSSI) solutions, and GSA Global Supply;
Learn more about federal environmental programs, such as WaterSense and Bio-Preferred, and
Discover optional environmental programs and additional procurement guidance that will help meet sustainability goals.
The green products portal identifies all product types covered by mandatory and non-mandatory federal environmental programs, including:
Bio-Preferred;
Comprehensive Procurement Guidelines;
Safer Choice;
Energy Star;
EPEAT;
Federal Energy Management Program (FEMP);
Significant New Alternatives Policy (SNAP); and
WaterSense.
In order to determine which requirements apply to the products planned for purchase, the portal allows users to search or browse for the product.
The services portal contains green purchasing guidance for services, including:
Cafeteria & Food Services;
Electronic Leasing;
Fleet Maintenance;
Janitorial Services;
Landscaping Services;
Laundry Services;
Meeting & Conference Services;
Pest Management; and
Transportation Services.